--cgroup-parent fails on systemd

sorry ping @vish

```
unexpected file format for /proc/self/cgroup - ""--- FAIL: TestRunContainerWithCgroupParent (0.10s)
	docker_cli_run_unix_test.go:137: unable to find self cpu cgroup path. CgroupsPath: map[blkio:/system.slice/docker-cd49b1e38a0c32aeaae5139402dd2a5a1b8bf10c77a04de1f054c8c4829558cb.scope net_cls:/ freezer:/system.slice/docker-cd49b1e38a0c32aeaae5139402dd2a5a1b8bf10c77a04de1f054c8c4829558cb.scope devices:/system.slice cpuset:/system.slice/docker-cd49b1e38a0c32aeaae5139402dd2a5a1b8bf10c77a04de1f054c8c4829558cb.scope perf_event:/ memory:/system.slice/docker-cd49b1e38a0c32aeaae5139402dd2a5a1b8bf10c77a04de1f054c8c4829558cb.scope cpuacct,cpu:/system.slice/docker-cd49b1e38a0c32aeaae5139402dd2a5a1b8bf10c77a04de1f054c8c4829558cb.scope name=butts:/system.slice/docker-cd49b1e38a0c32aeaae5139402dd2a5a1b8bf10c77a04de1f054c8c4829558cb.scope hugetlb:/]
unexpected file format for /proc/self/cgroup - 
```

--- a/integration-cli/docker_cli_run_unix_test.go
+++ b/integration-cli/docker_cli_run_unix_test.go
@@ -114,7 +114,7 @@ func getCgroupPaths(test string) map[string]string {
 	for _, line := range strings.Split(test, "\n") {
 		parts := strings.Split(line, ":")
 		if len(parts) != 3 {
-			fmt.Printf("unexpected file format for /proc/self/cgroup - %q", line)
+			fmt.Printf("unexpected file format for /proc/self/cgroup - %q\n", line)
 			continue
 		}
 		cgroupPaths[parts[1]] = parts[2]
@@ -132,7 +132,7 @@ func TestRunContainerWithCgroupParent(t *testing.T) {
 		t.Fatalf("failed to read '/proc/self/cgroup - %v", err)
 	}
 	selfCgroupPaths := getCgroupPaths(string(data))
-	selfCpuCgroup, found := selfCgroupPaths["cpu"]
+	selfCpuCgroup, found := selfCgroupPaths["memory"]
 	if !found {
 		t.Fatalf("unable to find self cpu cgroup path. CgroupsPath: %v", selfCgroupPaths)
 	}

